javascript/builtins/Object.json

```
@@ -819,7 +819,8 @@
                 "version_added": "4"
               },
               "ie": {
-                "version_added": "8"
+                "version_added": "9",
```

There **was** actually support in IE 8, it just wasn't great, so let's undo the change to this line.

```suggestion
                "version_added": "8",
```

---

Should i change defineProperty then, too?